articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,23 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
```

Comment:
This seems very vague. Can we be a bit more specific here, especially for a newcomer to K8S/AKS.

Reply:
I'll chat with my team on this, but this limitation already exists in the current doc: https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli#limitations

Reply:
cc @justindavies if you have any suggestions for rewording here?

Reply:
Applying custom configuration to the node kubelet is not supported